README.md

```diff
@@ -0,0 +1,27 @@
+# Mangrove [![Build Status](https://travis-ci.org/mongodb/mongo-cxx-odm.svg?branch=master)](https://travis-ci.org/mongodb/mongo-cxx-odm)
+
+Welcome to Mangrove, the official MongoDB C++ ODM!
+
+Mangrove lets you map your C++ classes to collections in MongoDB for seamless data retrieval and manipulation.
+
+#### Quickstart Guide
+
+The documentation for Mangrove is currently not hosted anywhere. You can generate the documentation on your own by running the `generate_docs.sh` script in this root directory. It requires that you have [Hugo](https://gohugo.io/) and [Doxygen](http://www.stack.nl/~dimitri/doxygen/) installed on your system. If you're on OS X, both can be installed with [Homebrew](http://brew.sh/).
```

Comment:
Yeah everything is static. I was originally thinking of just hosting them all on gh-pages but that works too.